src/lib/mina_commands/mina_commands.ml

```diff
@@ -47,6 +47,19 @@ let get_balance t (addr : Account_id.t) =
   let%map account = get_account t addr in
   account.Account.Poly.balance
 
+(* precomputed block writer *)
+let set_dump_dir ?network ~path t = Mina_lib.set_dump_dir ?network ~path t
```

Comment:
These would be much more self-documenting in a module `Precomputed_blocks` (so this becomes `Mina_commands.Precomputed_blocks.set_dump_dir` etc.)

Reply:
Agreed! Will change